dds/idl/v8_generator.cpp

```diff
@@ -168,6 +353,19 @@ namespace {
                  field->field_type(), prop.c_str());
     }
   };
+
+  struct gen_field_copyfrom {
+    gen_field_copyfrom(const char* tgt, const char* src) : tgt_(tgt), src_(src) {}
+    const std::string tgt_, src_;
+    void operator()(AST_Field* field) const
+    {
+      const std::string fieldName = field->local_name()->get_string(),
+        source = src_,
+        prop = fieldName;
```

Comment:
The `prop` variable seems to be unnecessary, just use `fieldName` directly.  Also the use of `source` can just be `src_`.

Reply:
Just blindly following the pattern of the other method. Turns out it doesn't make sense here, so I'll update it.